Gordon360/Services/HousingService.cs

```diff
@@ -34,36 +34,73 @@ public IEnumerable<HousingViewModel> GetAll()
 
         /// <summary>
         /// Saves student housing info
+        /// - first, it creates a new row in the applications table and puts the id of the primary applicant and a timestamp
+        /// - second, it looks for the application id of the application with the information we just input (because 
```

Comment:
Similar to the previous comment, it might be better to use "inserted" rather than input. The sentence itself makes sense though.

Reply:
Yeah, and it might make sense for me to say "it retrieves" instead of "it looks for" while I am at it